.travis.yml

```diff
@@ -8,3 +8,13 @@ rvm:
 
 cache: bundler
 script: bundle exec middleman build
+deploy:
+  provider: s3
+  access_key_id: $AWS_ACCESS_KEY
```

Comment:
@mikestonecodes -- If you can update this block with the below one, the PR is good to be merged and tested. Let us know once done. Thanks!

```
access_key_id: $AWS_ACCESS_KEY_ID
secret_access_key: $AWS_SECRET_ACCESS_KEY
bucket: $AWS_BUCKET_NAME
```